### DISCUSSION TITLE
Relationship to electric-sql?

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [electric-sql/pglite#28] [General]
Is there some vague aspirational plan to be able to use either sqlite or pgsql at some stage for electric, or is this definitely a parallel thing? I know Electric employees seem to have been given a "never confirm or deny anything about features we haven't implemented yet" (except regarding shapes...) from legal but it would be nice to know whether this is worth playing with in the near term!

**Comment by thruflo** [comment]
Yup, the idea is absolutely to use PGLite with Electric.

We’ve done quite a lot of work on supporting Postgres as the client database already, for the Postgres in Tauri approach. I believe that once PGLite supports parameterised queries then it will just work with Electric.

**Reply by AntonOfTheWoods** [reply]
That would be pure awesomeness. Well worth a look then!
